src/components/content/policies/Policies.tsx

```diff
@@ -0,0 +1,297 @@
+                                        deletePoliciesManagementServiceRequest.isError) &&
+                                    !isDeletingCompleted
+                                }
+                            >
+                                details
+                            </Button>
+                            <Button
+                                type='primary'
+                                icon={<EditOutlined />}
+                                onClick={() => {
+                                    updatePoliciesManagementService(record);
+                                }}
+                                disabled={
+                                    (deletePoliciesManagementServiceRequest.isPending ||
+                                        deletePoliciesManagementServiceRequest.isSuccess ||
+                                        deletePoliciesManagementServiceRequest.isError) &&
+                                    !isDeletingCompleted
+                                }
+                            >
+                                update
+                            </Button>
+                            <Popconfirm
+                                title='Delete the policy'
+                                description='Are you sure to delete the policy?'
+                                cancelText='Yes'
+                                okText='No'
+                                onCancel={() => {
+                                    deleteCurrentPolicy(record);
+                                }}
+                            >
+                                <Button
+                                    type='primary'
+                                    icon={<CloseCircleOutlined />}
+                                    loading={
+                                        record.id === id &&
+                                        !deletePoliciesManagementServiceRequest.isSuccess &&
+                                        !deletePoliciesManagementServiceRequest.isError
+                                    }
+                                    disabled={
+                                        (deletePoliciesManagementServiceRequest.isPending ||
+                                            deletePoliciesManagementServiceRequest.isSuccess ||
+                                            deletePoliciesManagementServiceRequest.isError) &&
+                                        !isDeletingCompleted
+                                    }
+                                >
+                                    delete
+                                </Button>
+                            </Popconfirm>
+                        </Space>
+                    </>
+                );
+            },
+        },
+    ];
+
+    const deleteCurrentPolicy = (record: PolicyVo) => {
+        setId(record.id);
+        setIsDeletingCompleted(false);
```

Comment:
setIsDeletingCompleted - we can try to use the state directly from the useMutation state.